recipe/build.sh

```diff
@@ -1,7 +1,14 @@
+if [[ "$(uname)" == "Darwin" ]]
+then
+    export CPPFLAGS=$(echo "${CPPFLAGS}" | sed -E "s@ \-mmacosx\-version\-min=${MACOSX_DEPLOYMENT_TARGET}@@g")
+    export CPPFLAGS="-mmacosx-version-min=${MACOSX_DEPLOYMENT_TARGET} ${CPPFLAGS}"
+fi
```

Comment:
Raised issue ( https://github.com/AnacondaRecipes/aggregate/issues/107 ) to address this more elegantly.